### PR TITLE
documentation

### PR DESCRIPTION
**Pull request by Rajitha04**
added documentation for avatar component

### CODE REVIEW

File: avatar.css [hunk 1]
```diff
@@ -109,4 +111,26 @@
...
+.avatar-container{
+    max-width: 500px;
+    margin: auto;
+}
+.heading-content{
+    text-align: center;
+}
+.para-content{
+    text-align: center;
+    font-size: 20px;
```
Comment (anupkrbid) [inline avatar.css:124]: Prefer using rem over px and vh.

File: avatar.html [hunk 2]
```diff
@@ -22,9 +27,19 @@
...
+
+<iframe
+src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar-small%2522%253EAA%253C%252Fdiv%253E%2520%2520%2520%2520%250A%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar-medium%2522%253EBB%253C%252Fdiv%253E%2520%2520%2520%2520%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar-large%2522%253EVJ%253C%252Fdiv%253E%2520%2520%2520%2520%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522avatar-xtralarge%2522%253ERJ%253C%252Fdiv%253E%2520%2520%2520%2520%250A%253C%252Fdiv%253E"
+style="width: 538px; height: 409px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (anupkrbid) [inline avatar.html:36]: Avoide using inline styles. You can move this to your css file and use the class name

File: avatar.html [hunk 3]
```diff
@@ -59,6 +81,13 @@
...
+
+<iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar-small-square%2522%2520src%253D%2522avatar1.jpg%2522%252F%253E%2520%2520%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar-medium-square%2522%2520src%253D%2522avatar1.jpg%2522%252F%253E%2520%2520%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar-large-square%2522%2520src%253D%2522avatar1.jpg%2522%252F%253E%2520%2520%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar-xtralarge-square%2522%2520src%253D%2522avatar1.jpg%2522%252F%253E%2520%2520%250A%253C%252Fdiv%253E"
+  style="width: 554px; height: 484px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (anupkrbid) [inline avatar.html:90]: Same

File: avatar.html [hunk 4]
```diff
@@ -42,10 +57,17 @@
...
+  style="width: 554px; height: 540px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (anupkrbid) [inline avatar.html:63]: Same

**Review by anupkrbid — COMMENTED** [review]
Over all looks good. Just prefer using rem over px and vh as much as possible. Px is absolute and vh won't behave correctly when you zoom the browser.

File: avatar.html (hunk 4 above)
Comment (Rajitha04) [reply avatar.html:63]: sure will make the changes

**Comment by SreevidyaNattuva** [comment]
image
Could have used some class instead of mentioning text-align center more than twice

**Pull request merged**